src/v2/view-builder/components/OdaOktaVerifyTerminalView.js

```diff
@@ -72,3 +85,76 @@ export default View.extend({
     });
   },
 });
+
+const AndroidAppLinkWithAccountOdaTerminalView =  BaseOdaOktaVerifyTerminalView.extend({
+  template: hbs`
+    <p class="explanation" data-se="subheader">
+    {{i18n code="enroll.oda.with.account.explanation" bundle="login"}}
+    </p>
+    <p class="subtitle">
+    {{i18n code="enroll.oda.with.account.subtitile1 " bundle="login"}}
+    </p>
+    <ul>
+      <li>{{i18n code="enroll.oda.with.account.step1" bundle="login"}}</li>
+      <li>{{i18n code="enroll.oda.with.account.step2" bundle="login"}}</li>
+      <li>{{i18n code="enroll.oda.with.account.step3" bundle="login"}}</li>
+    </ul>
+    <p class="subtitle">
+    {{i18n code="enroll.oda.with.account.subtitile2 " bundle="login"}}
+    </p>
+    <ol>
+      <li>{{i18n code="enroll.oda.with.account.step4" bundle="login"}}</li>
+      <li>{{i18n code="enroll.oda.with.account.step5" bundle="login" arguments="orgName"}}</li>
+      <li>{{i18n code="enroll.oda.with.account.step6" bundle="login"}}</li>
+      <li>{{i18n code="enroll.oda.with.account.step7" bundle="login"}}</li>
+    </ol>
+  `,
+});
+
+const AndroidAppLinkWithoutAccountOdaTerminalView =  BaseOdaOktaVerifyTerminalView.extend({
+  template: hbs`
+    <p class="explanation" data-se="subheader">
+      {{i18n code="enroll.oda.without.account.explanation" bundle="login"}}
+    </p>
+    <ol>
+      <li>
+        <p>{{i18n code="enroll.oda.without.account.step1" bundle="login"}}</p>
+        <p><a href="{{appStoreLink}}" target="_blank" id="download-ov">
+          {{i18n code="enroll.oda.without.account.download.app" bundle="login"}}
+        </a></p>
+      </li>
+      <li>{{i18n code="enroll.oda.without.account.step2" bundle="login"}}</li>
+      <li>
+        {{i18n code="enroll.oda.step3" bundle="login"}}
+        <p class="org-signin-link">
+          <span class="no-translate">
+          {{signInUrl}}
+          </span>
+        </p>
+        <a data-clipboard-text="{{signInUrl}}" class="button link-button copy-org-clipboard-button">
+          {{i18n code="enroll.oda.org.copyLink" bundle="login"}}
+        </a>
+      </li>
+      <li>{{i18n code="enroll.oda.without.account.step4" bundle="login"}}</li>
+    </ol>
+  `,
+
+  postRender: function() {
+    // Attach each button to the respective 'data-clipboard-text'
+    Clipboard.attach('.copy-org-clipboard-button').done(() => {
```

Comment:
the button style seems to a little different. The mocks is using a button that spread to the whole line.